src/_modules/experimental/Request.sol

```diff
@@ -356,17 +376,26 @@ function Ok(Request memory value) pure returns (RequestResult) {
     assembly {
         _value := value
     }
-    return RequestResult.wrap(Pointer.unwrap(ResultType.Ok.encode(_value)));
+    return ResultType.Ok.encode(_value).toRequestResult();
 }
 
 function Ok(Response memory value) pure returns (ResponseResult) {
     bytes32 _value;
```

Comment:
Maybe these could be `Pointer _value`? not really necessary but it makes sense for `encode` to receive a Pointer

Reply:
Added some more simplifications here cd39646